homeassistant/components/history/__init__.py

```diff
@@ -62,375 +51,60 @@
     extra=vol.ALLOW_EXTRA,
 )
 
-SIGNIFICANT_DOMAINS = (
-    "climate",
-    "device_tracker",
-    "humidifier",
-    "thermostat",
-    "water_heater",
-)
-IGNORE_DOMAINS = ("zone", "scene")
-NEED_ATTRIBUTE_DOMAINS = {
-    "climate",
-    "humidifier",
-    "input_datetime",
-    "thermostat",
-    "water_heater",
-}
-
-QUERY_STATES = [
-    States.domain,
-    States.entity_id,
-    States.state,
-    States.attributes,
-    States.last_changed,
-    States.last_updated,
-]
-
-HISTORY_BAKERY = "history_bakery"
-
 
 def get_significant_states(hass, *args, **kwargs):
```

Comment:
We have a deprecated_function wrapper.

https://github.com/home-assistant/core/blob/dev/homeassistant/util/__init__.py#L50